grafana-plugin/src/containers/UserSettings/parts/tabs/PersonalWebhookInfo/PersonalWebhookInfo.tsx

```diff
@@ -159,29 +155,50 @@ export const PersonalWebhookInfo = observer(() => {
               <Select
                 {...field}
                 menuShouldPortal
-                placeholder={(user.messaging_backends?.WEBHOOK?.name as string) ?? 'Select personal webhook to trigger'}
+                placeholder={(user.messaging_backends?.WEBHOOK?.name as string) ?? 'Select webhook'}
                 onChange={({ value }) => field.onChange(value)}
                 options={webhookOptions}
               />
             }
           />
         </Field>
-        {selectedWebhook != null ? (
-          <Field
-            label="Context"
-            description="Optional. Set additional context (as a JSON dict) to make it available in the webhook payload and/or templates."
-            invalid={!!errors.context}
-            error={errors.context?.message}
+        {
+          selectedWebhook != null ? (
+            <Field
+              label="Context"
+              description={
+                <p>You can add additional JSON template to use the same webhook for multiple users events.
+                  <a href={"https://grafana.com/docs/oncall/latest/configure/integrations/outgoing-webhooks"} target="_blank" rel="noreferrer" className={styles.link}>
```

Comment:
We can update this href to something more specific to this use-case if it exists.

Reply:
My PR is adding a specific [docs page](https://github.com/grafana/oncall/pull/5426/files#diff-c33c9020f77fbedfcd5db47961edc0d0ff0774b712a3ec10736f558879518e39) in `https://grafana.com/docs/oncall/latest/manage/notify/webhook/`

Also, I would change the description to be more like:
"You can add additional JSON context to be used by the webhook templates allowing for webhook reusability between different users."

Reply:
Yes, that makes sense. TBH, I copied it straight out of Figma without reading it, I should have checked. I'll update that and the docs link.